assets/src/components/detours/diversionPage.tsx

```diff
@@ -245,6 +245,17 @@ export const DiversionPage = ({
             />
           ) : snapshot.matches({ "Detour Drawing": "Active" }) ? (
             <ActiveDetourPanel
+              directions={extendedDirections}
+              connectionPoints={[
+                connectionPoints?.start?.name ?? "N/A",
+                connectionPoints?.end?.name ?? "N/A",
+              ]}
+              missedStops={missedStops}
+              routeName={routeName ?? "??"}
+              routeDescription={routeDescription ?? "??"}
+              routeOrigin={routeOrigin ?? "??"}
+              routeDirection={routeDirection ?? "??"}
+              onNavigateBack={() => onConfirmClose}
```

Comment:
```suggestion
              onNavigateBack={onClose}
```

In order to make the back button behave the same as the close button.

Reply:
We don't want it to behave the same as the close button. For an active detour, we don't need the confirm close modal to appear

Reply:
Oh yeah I guess the confirm dialog isn't really necessary from the active state.

In that case:
```suggestion
              onNavigateBack={onConfirmClose}
```